spec/services/big_blue_button_api_spec.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'bigbluebutton_api'
+
+describe BigBlueButtonApi, type: :service do
+  let(:bbb_server) { described_class.new }
+
+  describe 'Instance of BigBlueButtonApi created' do
+    it 'bbb_server is created and an instance of BigBlueBUttonApi' do
+      expect(bbb_server).to be_instance_of(described_class)
+    end
+  end
```

Comment:
This test doesn't actually test anything except Rails functionality.

Essentially what this test is doing is saying the BigBlueButtonApi.new returns an instance of BigBlueButtonApi

Reply:
We could make the `bbb_server` method public and then test that it initializes an instance of BigBlueButton::BigBlueButtonApi with the right params passed in